Modules/SubModule_Import/xASL_bids_BIDS2Legacy.m

```diff
@@ -97,21 +96,23 @@
 
         %% 3. Define Session
         iVisit = find(strcmp(BIDS.sessionName, SessionID));
-        % remove iteration for iVisit=1 % iterate visit/session in this "BIDS.subjects" (always 1 session per BIDS.subjects)
+        % Remove iteration for iVisit = 1 -> iterate visit/session in this "BIDS.subjects" (always 1 session per BIDS.subjects)
         % ExploreASL uses visit as a number (e.g. _1 _2 _3 etc)
         if nVisits==1
-            pathLegacy_SubjectVisit = fullfile(pathLegacy, SubjectID);
+            pathLegacy_SubjectVisit = fullfile(x.dir.pathLegacy, SubjectID);
```

Comment:
If we have `x.dir.DatasetRoot`, would make it more consistent to also have `x.dir.raw` and `x.dir.derivatives`? `x.dir.pathLegacy` is a new name, not clear to the new reader, but pathDerivatives or something similar is clear (and this is what we mean here right?

Reply:
Very good comment. `DatasetRoot` is good for the root to all directories. `RawData`, `SourceData`, `Derivatives` might be used as well. So while we are at it, we can probably change `pathLegacy` to `Derivatives`, but it can't be just derivatives - that would preclude further extensions and `xASLDerivatives` might be better - just to show that we don't mean root derivatives folder. In general, regarding use of `Legacy` - apart from this case, we don't need to rename it that much, because all Legacy will be later renamed to fit better the BIDS derivatives format.

Reply:
Done